server/routes.js

```diff
@@ -113,12 +113,25 @@ export default async app => {
 
   // oAuth server (after authentication/JWT handling, at least for authorize)
   app.oauth = oauth;
-  app.post('/oauth/token', app.oauth.token());
+  app.post('/oauth/token', noCache, app.oauth.token());
   app.post(
     '/oauth/authorize',
+    noCache,
     app.oauth.authorize({ allowEmptyState: true, authenticateHandler: authorizeAuthenticateHandler }),
   );
-  app.post('/oauth/authenticate', app.oauth.authenticate());
+  app.post('/oauth/authenticate', noCache, app.oauth.authenticate());
+  app.get('/oauth/me', noCache, async (req, res) => {
+    if (req.remoteUser) {
```

Comment:
Shouldn't we reply with a 404 if there's no `remoteUser`?

Reply:
Maybe, I'm gonna remove this thing for now, will potentially re-implement it differently.